modules/grafana/files/dashboards/sidekiq.json

```diff
@@ -142,7 +144,7 @@
           "targets": [
             {
               "refId": "A",
-              "target": "summarize(groupByNode(stats.gauges.govuk.app.$Application.*.workers.queues.*.latency, 8, 'avg'), '$Aggregation', 'avg', false)"
+              "target": "aliasByNode(summarize(averageSeriesWithWildcards(stats.gauges.govuk.app.$Application.*.workers.queues.*.latency, 5), '$Aggregation', 'avg', false), 7)"
```

Comment:
These changes relate to the queue latency, not the new panels. Were you intending to make them?

Reply:
Oh yes, sorry. I noticed this panel was doing a weird `groupByNode` that was causing the legend to come out in the Graphite expression syntax. I fixed it to resemble the other queries we do in the dashboard. The datapoints should be unchanged, although this is Graphite, so maybe `groupByNode(..., 'avg')` is subtly different to `averageSeriesWithWildcards(...)`!